src/System.Diagnostics.Debug/tests/System.Diagnostics.Debug.Tests.csproj

```diff
@@ -30,6 +30,7 @@
     </ProjectReference>
     <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj" />
     <ProjectReference Include="..\..\System.Runtime.Extensions\src\System.Runtime.Extensions.csproj" />
+    <TargetingPackReference Include="System.Private.CoreLib" Condition="'$(TestTFM)' != 'net46'" />
```

Comment:
The white-box part of the tests could not ever work on net46. I have excluded them for TargetGroup net46 configuration.